project_euler/problem_81/sol1.py

```diff
@@ -36,6 +36,7 @@ def dijkstra(initial,graph):
 
 with open('input.txt') as in_file:
     matrix = [[int(x) for x in line.split(',')] for line in in_file]
+in_file.close()
                                          
```

Comment:
__with open() as__ removes the need to call __.close()__.

Also one line was deleted in these changes that needs to be re-added.

Reply:
oh yess i hope its good now...😉

Reply:
Nice!  Thanks.